style.css

```diff
@@ -0,0 +1,194 @@
+
+* {
+    box-sizing: border-box;
+}
+
+html, body, div, img {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline
+}
+body {
+    line-height: 0,5;
+    background-color: #242424;
+    font-family: outfit;
+    align-content: center;
+}
+
+p{
+  margin:9px;
+}
+
+
+}
+.main-header{
+    display: flex
+    flexbox: flex-end;
+	flex-wrap: wrap;
+  justify-content: space-around;
+    flex-direction: column-reverse;
+    background-color:#242424;
+    align-items: center;
+}
+
+@media (min-width:1024px){
+  .main-header{
+    flex-direction: column-reverse;
+  }
+}
+
+
+img {
+    align-items: center;
+
+}
+
+.sitename{
+text-align:center;
+color:rgb(248, 248, 248);
+}
+.h1{
+  font-size: 55pt;
+  margin: -5px;
+}
+
+.navbar{
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly
+}
+ul{
+    list-style: none;
+    overflow: hidden;
+    font-size: 14pt;
+    justify-content: flex-end;
+    margin-bottom: 10pt;
+}
+
+li {
+    float:left;
+    display: inline;
+    padding:2px;
+    font-style: italic;
+    font-weight: bolder;
+  }
+  li a {
+    display: block;
+    color: white;
+    text-align:right;
+    padding:10pt;
+    text-decoration: none;
+    font-weight: 300;
+   background-color: #F4E139;
+   border-radius: 2%;
+  }
+
+  li a:hover {
+    background-color: #06767B;
+  }
+
+.sitename{
+  align-items: flex-end;
+}
+
+  .main-article{
+    display:flex;
+    flex-direction: column;
+    background-color: #06767B;
+    width: 100%;
+ 
+}  
+
+.article-pic {  
+max-width: 100%;  
+height: 500px;
+width: 800px;
+align-items: center;
+  }  
+
+  .djokovic{
+      display: flex;
+      flex-direction: column;
+      margin-left: 10px;
+      margin-right: 10px;
+  
+  }
+
+  .heading{
+    font-size: 20pt;
+   
+  }
+  .article-text{
+    font-size: 14pt;
+  padding: 5pt;
+  }
+
+  img {
+    width:  320px;
+    height: 320px;
+    object-fit:cover;
+  }
+
+
+  .cardbox{
+    display: flex;
+    align-items:center;
+    margin: 20px;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    flex-direction: column;
+    background-color:#242424;
+  }
+
+  .card{
+    height:400px;
+    width: 320px;
+    padding:10px;
+
+    background-color: #06767B;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    text-align: center;
+  }
+
+  .card:hover{
+    background-color: #F4E139;
+  }
+
+  @media (min-width: 667px) and (max-width: 1024px){
```

Comment:
I think here you are missing the transition from having a logo and the name of the page (desktop) to having just the logo (tablet) - see sketch of the assignment.

Reply:
I am reading the project specs again now and perhaps I am wrong here and this wasn't a fixed requirement?